src/videodownlinkbandwidthpolicy/VideoDownlinkBandwidthPolicy.ts

```diff
@@ -54,6 +56,11 @@ export default interface VideoDownlinkBandwidthPolicy {
    */
   removeObserver?(observer: VideoDownlinkObserver): void;
 
+  /**
+   * (Optional) Call and observer function on all added `VideoDownlinkObserver`
+   */
+  forEachObserver?(observerFunc: (observer: VideoDownlinkObserver) => void): void;
```

Comment:
Why do we need to add this to the interface? Do you call it somewhere outside of the video priority class?

Reply:
Oops that was for future change, will remove.